utils/filter.c

```diff
@@ -38,72 +38,131 @@ static void snprintf_trigger_read(char *buf, size_t len, enum trigger_read_type
 
 static void print_trigger(struct uftrace_trigger *tr)
 {
-	if (tr->flags & TRIGGER_FL_CLEAR)
-		pr_dbg("\ttrigger: clear\n");
-	if (tr->flags & TRIGGER_FL_DEPTH)
-		pr_dbg("\ttrigger: depth %d\n", tr->depth);
+	if (tr->flags & TRIGGER_FL_DEPTH) {
+		if (tr->clear_flags & TRIGGER_FL_DEPTH)
+			pr_dbg("\tclear trigger: depth\n");
+		else
+			pr_dbg("\ttrigger: depth %d\n", tr->depth);
+	}
```

Comment:
I'm not sure if we need this whole changes.  Can we simply do like below and leave other code as is?

```
if (tr->flags & TRIGGER_FL_CLEAR)
    pr_dbg("\ttriggers: clear=%x\n", tr->clear_flags);
```